<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_addRow">
        <property name="text">
         <string>Add row</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_deleteRow">
        <property name="text">
         <string>Delete row</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_recent_file">
     <property name="title">
      <string>Open recent file</string>
     </property>
     <addaction name="actionRecentFile0"/>
     <addaction name="actionRecentFile1"/>
     <addaction name="actionRecentFile2"/>
     <addaction name="actionRecentFile3"/>
     <addaction name="actionRecentFile4"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_file"/>
    <addaction name="menuOpen_recent_file"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionSave_all"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionNumActiveWindow"/>
    <addaction name="separator"/>
    <addaction name="actionClose_all_windows"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose_all_windows">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close all windows</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New file</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save all</string>
   </property>
  </action>
  <action name="actionNumActiveWindow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>(none)</string>
   </property>
   <property name="toolTip">
    <string>(none)</string>
   </property>
  </action>
  <action name="actionRecentFile0">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>(none)</string>
   </property>
   <property name="toolTip">
    <string>(none)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRecentFile1">
   <property name="text">
    <string>(none)</string>
   </property>
  </action>
  <action name="actionRecentFile2">
   <property name="text">
    <string>(none)</string>
   </property>
  </action>
  <action name="actionRecentFile3">
   <property name="text">
    <string>(none)</string>
   </property>
  </action>
  <action name="actionRecentFile4">
   <property name="text">
    <string>(none)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
